utils/dom.js

```diff
@@ -395,6 +395,40 @@ export function documentHasSelection() {
 	return range && ! range.collapsed;
 }
 
+/**
+ * Check wether the contents of the element have been fully selected.
+ * Returns true if there is no possibility of full selection.
+ *
+ * @param {Element} element The element to check.
+ *
+ * @return {boolean} True if fully selected, false if not.
+ */
+export function isFullySelected( element ) {
```

Comment:
I might name this "entirely" rather than "fully" unless there's a history of using "fullySelected". "entirely" just seems a bit clearer and more explicit as to what this means.